recipes/conda/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
```

Comment:
If I remember correctly, you also need to always_include the conda executable: http://conda.pydata.org/docs/building/meta-yaml.html#force-files.